.github/workflows/performance.yml

```diff
@@ -4,18 +4,11 @@ on:
   push:
     branches:
       - trunk
-      - 'feature/**'
       - '3.[7-9]'
       - '[4-9].[0-9]'
     tags:
       - '[0-9]+.[0-9]'
       - '[0-9]+.[0-9].[0-9]+'
-  pull_request:
-    branches:
-      - trunk
-      - 'feature/**'
-      - '3.[7-9]'
-      - '[4-9].[0-9]'
```

Comment:
These need to stay, just remove the feature branch part.

Reply:
Thanks @joemcgill, The changes reverted in https://github.com/10up/wordpress-develop/pull/41/commits/e66687a08628d2348048dba0ec6cddc19a6d2937